src/main/kotlin/no/skatteetaten/aurora/boober/feature/fluentbit/FluentbitSidecarFeature.kt

```diff
@@ -150,7 +150,7 @@ class FluentbitSidecarFeature(
                         "$FEATURE_FIELD_NAME/custom/$key/pattern",
                         validator = {
                             it.pattern(
-                                "^[A-Za-z-]+\\.[A-Za-z]+\$",
+                                "^[A-Za-z-*]+\\.[A-Za-z]+\$",
```

Comment:
Patternet bør kanskje legges i en variabel som brukes her og i feilmeldingen, det er mismatch mellom de to nå

Reply:
Lurt! Oppdatert nå og startet bygg